run.py

```diff
@@ -986,13 +1029,26 @@ def engine_manifest():
         cancellable_engine = CancellableEngine(args)
 
     root_dir = args.voicevox_dir if args.voicevox_dir is not None else engine_root()
+
+    settings = SettingLoader(root_dir / "setting.yml").load_setting_file()
```

Comment:
ここでこうするといいのかなと！
（あとこの上で作られてるroot_dirはエンジンのrootじゃないときがあるので、`engine_root()`をそのまま指定するのが正しそうです！）
```suggestion
    setting_loader = SettingLoader(engine_root() / "setting.yml")
    settings = setting_loader.load_setting_file()
```